src/main/java/in/projecteka/consentmanager/common/TokenService.java

```diff
@@ -1,4 +1,4 @@
-package in.projecteka.consentmanager.user;
+package in.projecteka.consentmanager.common;
```

Comment:
Ideally tokenForAdmin in this method can be removed, and all the places wherever we are using that method we could use , identityservice.authenticate method.

Reply:
Can be separate PR too:)

Reply:
I will create separate PR for this